packages/assistify-ai/config.js

```diff
@@ -77,3 +77,26 @@ Meteor.startup(() => {
 		RocketChat.settings.get('Assistify', addAISettings) :
 		RocketChat.settings.addGroup('Assistify', addAISettings);
 });
+
+RocketChat.settings.get('Assistify_AI_Smarti_Base_URL', (id, smartiValue) => {
+	const domain = RocketChat.models.Settings.findOneNotHiddenById('Assistify_AI_Smarti_Domain');
+	if (domain) {
+		RocketChat.models.Settings.update({_id: 'CHATPAL_BASEURL'}, {
+			$set: {
+				value: `${ smartiValue }rocket/${ domain.value }/search-message`,
+				readonly: true,
+				enableQuery: '{"_id":"Assistify_AI_Smarti_Base_URL","value":""}'
```

Comment:
This disables the input field if the value is effectively copied from the Assistify-setting